packages/sit-onyx/src/components/OnyxDataGrid/OnyxDataGrid.stories.ts

```diff
@@ -23,3 +27,51 @@ export const Default = {
     ],
   },
 } satisfies Story;
+
+export const MoreActions = {
+  args: {
+    columns: ["name", "age", "birthday"],
+    features: [
+      {
+        name: Symbol("More actions"),
+        watch: [],
+        header: {
+          actions: (column) =>
+            column === "name"
+              ? [
+                  {
+                    iconComponent: h(OnyxIconButton, {
+                      label: "More actions",
+                      icon: sort,
+                      color: "neutral",
+                    }),
+                    listItems: [
+                      h(OnyxListItem, () => "No sorting"),
+                      h(OnyxListItem, () => "Random sort"),
+                    ],
+                  },
+                  {
+                    iconComponent: h(OnyxIconButton, {
+                      label: "More actions",
+                      icon: sort,
+                      color: "neutral",
+                    }),
```

Comment:
Please use examples that don't use the label "More actions", otherwise that could be confusing :)